### DISCUSSION TITLE
How to transfer a website from port 3000 to port 80?

### DISCUSSION THREAD

**Discussion by AeShevch** [WebDevStudios/nextjs-wordpress-starter#801] [Q&A]
Greetings!
Please tell me how to make the frontend work on a standard port?
I tried to run through `npm start -p 80`, but I get a cyclic redirect

**Comment by gregrickaby** [comment]
@AeShevch I found some documentation on this: https://nextjs.org/docs/api-reference/cli#production

**Comment by itsamoreh** [comment]
@AeShevch Instead of running `npm start -p 80`, edit the `start` command in `frontend/package.json`. Change it from `next start` to `next start -p 80`. Then try running `npm start`. This should work as long as you don't have anything running on port `80`.
